cypress/e2e/mediashelf/useMediaShelf.feature

```diff
@@ -1,10 +1,13 @@
-@regression_test
-@stable_test
+# this feature marked unstable, when dragging the element was detached
+# from DOM and cypress is failing the test. In GUI mode, you have to manually
+# show the path to the element and then it works but in headless mode thats
+# not the case and it fails every time.
+
+@unstable_test
```

Comment:
Is there a plan how to fix this? Follow-up ticket for capcakes?

Reply:
follow up ticket for caps will be discussed in weekly

https://ticketsystem.dbildungscloud.de/browse/BC-9322